src/components/PrimaryPanes/Outline.js

```diff
@@ -41,7 +43,18 @@ type Props = {
   flashLineRange: Function
 };
 
-export class Outline extends Component<Props> {
+type State = {
+  filter: string
+};
+
+export class Outline extends Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+    (this: any).filterOutlineItems = this.filterOutlineItems.bind(this);
+    (this: any).onUpdateFilter = this.onUpdateFilter.bind(this);
```

Comment:
If we use the ES6 arrow function definitions for `filterOutlineItems` and `onUpdateFilter` we wouldn't need to bind them here.  Do you want to do this?  If so do you want to change all the function definitions in the class to this style for consistancy?